pt/news/_posts/2016-11-09-ruby-2-4-0-preview3-released.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: news_post
+title: "Lançado o Ruby 2.4.0-preview3"
+author: "naruse"
+translator: "Filipe Rocha"
+date: 2016-11-09 18:00:00 +0900
+lang: pt
+---
+
+Temos o gosto de anunciar a versão 2.4.0-preview3 do Ruby.
+
+A versão 2.4.0-preview3 do Ruby é a terceira *preview* do Ruby 2.4.0,
+lançada com o fim de obter *feedback* da comunidade.
+
+Poderão
+[enviar *feedback*](https://bugs.ruby-lang.org/projects/ruby/wiki/HowToReport)
+uma vez que ainda podem influenciar das funcionalidades.
+
+## [Introdução de melhorias nas hash tables por Vladimir Makarov](https://bugs.ruby-lang.org/issues/12142)
+
+Melhorar a estrutura interna da hash table (st_table) introduzindo o *open addressing*
+e introdução de um *array* com a ordem de inclusão.
+
+## Binding#irb: Iniciar uma sessão REPL como `binding.pry`
+
+Enquanto está a fazer o *debug*, poderá utilizar `p` para listar o valor das variáveis.
+Com o [pry](https://github.com/pry/pry) e incluindo `binding.pry` na sua aplicação,
+será lançado este REPL para correr qualquer código ruby.
+[r56624](https://github.com/ruby/ruby/commit/493e48897421d176a8faf0f0820323d79ecdf94a) introduz `binding.irb` com o mesmo comportamento para o irb.
+
+## [Unificar as classes Fixnum e Bignum na classe Integer](https://bugs.ruby-lang.org/issues/12005)
+
+Apesar do [ISO/IEC 30170:2012](http://www.iso.org/iso/iso_catalogue/catalogue_tc/catalogue_detail.htm?csnumber=59579)
+não especificar os detalhes da classe Integer,
+o Ruby teve duas classes Integer visíveis: Fixnum and Bignum.
+O Ruby 2.4 unifica-as na classe Integer.
+Todas as extensões C que utilizam a classe Fixnum ou Bignum necessitam de ser corrigidas.
+
+Consultar também [o ticket](https://bugs.ruby-lang.org/issues/12005) e os [slides do akr](http://www.a-k-r.org/pub/2016-09-08-rubykaigi-unified-integer.pdf).
```

Comment:
I think `Consulte` goes better here. I'm not sure if you use infinitive form for this sentences in Portugal.